### ISSUE TITLE
Strategy Room Ship List: Bring back totals

### ISSUE THREAD

**Issue by dragonjet** [dragonjet/KC3Kai#259]
*ah, i kinda missed the info for ships total fp, tp, level, hp and etc. since i keep comparing it to my friend for fun* - originally from #253

**Comment by LostLogia4** [comment]
How about the average for all ships?

**Comment by bamboo3250** [comment]
Yes, I also think average of all ships makes more sense than just adding up together.

**Comment by LostLogia4** [comment]
Why not both?

**Comment by bamboo3250** [comment]
I have no objection to have both but it is just the problem of where to put the stats and the toggle button.
